src/components/Goban.js

```diff
@@ -262,9 +262,34 @@ export default class Goban extends Component {
         board.height
       )
 
+    // Calculate coordinates
+
+    let relativeCoord = (x, size) => {
+      let halfSize = Math.ceil(size / 2)
+      let ix = size - x + 1
+      if (x <= halfSize && x < 10) return x.toString()
+      else if (ix < halfSize && ix < 10) return `${ix}*`
+      else return 'X'
+    }
+
+    let getCoordTransformer = coordinatesType => {
```

Comment:
This variable might better be called `getCoordFunctions` since it might get confused with `transformCoords()`.